src/loan-comparison/index.html

```diff
@@ -1,5 +1,6 @@
 {% extends "base.html" %}
 {% set active_page = "loan-comparison" %}
+{% set page_js = "loan-comparison" %}
```

Comment:
I know you put this here, but we have commented out the loan-comparison.js.  Also if you want to add page_js, you should also add it to prepare-worksheets.js